lib/commands.sh

```
@@ -328,64 +366,412 @@ cmd_policy() {
+}
+
+cmd_setup() {
+  ensure_in_repo
+  resolve_policy
+
+  local strictness="${SHIPLOG_SETUP_STRICTNESS:-}"
+  local strict_envs_raw="${SHIPLOG_SETUP_STRICT_ENVS:-}"
+  local auto_push="${SHIPLOG_SETUP_AUTO_PUSH:-0}"
+  local authors_input="${SHIPLOG_SETUP_AUTHORS:-}"
+  local dry_run="${SHIPLOG_SETUP_DRY_RUN:-0}"
+  local include_self=1
```

Shellcheck (0.11.0)

[warning] include_self appears unused. Verify use (or export if used externally).

(SC2034)